source/extensions/common/wasm/wasm.h

```diff
@@ -844,6 +850,21 @@ inline bool Wasm::copyToPointerSize(const Buffer::Instance& buffer, uint64_t sta
   return true;
 }
 
+inline PluginDirection
+pluginDirectionFromTrafficDirection(envoy::api::v2::core::TrafficDirection direction) {
+  switch (direction) {
+  case envoy::api::v2::core::TrafficDirection::UNSPECIFIED:
+    return PluginDirection::Unspecified;
+  case envoy::api::v2::core::TrafficDirection::INBOUND:
+    return PluginDirection::Inbound;
+  case envoy::api::v2::core::TrafficDirection::OUTBOUND:
+    return PluginDirection::Outbound;
+  default:
+    ASSERT(!"Bad envoy::api::v2::core::TrafficDirection");
```

Comment:
Nit: you can remove this `ASSERT`, since `NOT_REACHED_GCOVR_EXCL_LINE` logs `"not reachable"` and calls `abort()`.